tests/python/shared/tasks/base.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2025 Intel Corporation
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from typing import Any, Union
+
+import attrs
+from cvat_sdk.api_client import models
+
+from .interface import ITaskSpec
+from .utils import parse_frame_step
+
+
+@attrs.define
+class _TaskSpecBase(ITaskSpec):
```

Comment:
leading _ underscore should be removed here and in other places where you expect a name to be imported in another module